build/dependencies.props

```diff
@@ -8,11 +8,12 @@
     <BenchmarkDotNetPackageVersion>0.10.13</BenchmarkDotNetPackageVersion>
     <InternalAspNetCoreSdkPackageVersion>2.1.3-rtm-15811</InternalAspNetCoreSdkPackageVersion>
     <MicrosoftAspNetCoreBenchmarkRunnerSourcesPackageVersion>2.2.0-preview1-34576</MicrosoftAspNetCoreBenchmarkRunnerSourcesPackageVersion>
+    <MicrosoftAspNetCoreAppPackageVersion>2.1.1</MicrosoftAspNetCoreAppPackageVersion>
```

Comment:
This version is used in our samples and test projects. Due to the issues with versioning and `Microsoft.AspNetCore.Razor.Design` we need to keep these roughly in sync with Razor.

Note that this version **does not** get baked into our packages. Those reference 2.1.0 versions of the ASP.NET Core packages they need.